### PR TITLE
CLDC-3000 Display available from date in the error message

### PR DESCRIPTION
**Pull request by kosiakkatrina**

### CODE REVIEW

File: spec/models/validations/setup_validations_spec.rb [hunk 1]
```diff
@@ -136,7 +136,7 @@
         Timecop.return
       end
 
-      let(:absorbing_organisation) { create(:organisation, created_at: Time.zone.local(2023, 2, 1, 4, 5, 6), available_from: Time.zone.local(2023, 2, 1, 4, 5, 6), name: "Absorbing org") }
+      let(:absorbing_organisation) { create(:organisation, created_at: Time.zone.local(2023, 1, 30, 4, 5, 6), available_from: Time.zone.local(2023, 2, 1, 4, 5, 6), name: "Absorbing org") }
       let(:absorbing_organisation_2) { create(:organisation, created_at: Time.zone.local(2023, 2, 1), available_from: Time.zone.local(2023, 2, 1), name: "Absorbing org 2") }
```
Comment (natdeanlewissoftwire) [inline spec/models/validations/setup_validations_spec.rb]: should we change the created_at here too for clarity?

File: app/models/validations/setup_validations.rb [hunk 2]
```diff
@@ -150,7 +150,7 @@ def add_same_merge_organisation_error(record)
     elsif absorbing_owning_organisation_inactive?(record)
       record.errors.add :startdate, I18n.t("validations.setup.startdate.invalid_absorbing_organisations_start_date.same_organisation",
                                            owning_organisation: record.owning_organisation.name,
-                                           owning_organisation_available_from: record.owning_organisation.created_at.to_formatted_s(:govuk_date))
+                                           owning_organisation_available_from: record.owning_organisation.available_from.to_formatted_s(:govuk_date))
```
Comment (natdeanlewissoftwire) [inline app/models/validations/setup_validations.rb:153]: is there any risk of available_from not existing here?

**Review by natdeanlewissoftwire — APPROVED** [review]

File: app/models/validations/setup_validations.rb (hunk 2 above)
Comment (kosiakkatrina) [reply app/models/validations/setup_validations.rb:153]: `absorbing_owning_organisation_inactive?` checks that `record.owning_organisation.available_from.present?`, so should be all good!

File: spec/models/validations/setup_validations_spec.rb (hunk 1 above)
Comment (kosiakkatrina) [reply spec/models/validations/setup_validations_spec.rb]: Sure!

**Pull request merged**